Nixtla/mlforecast
# Add Mintlify automations

## Description
This PR adds the automations necessary to create the statsforecast Mintlify docs. When a push to `main` is made, the `build-docs` workflow will run which will upload the needed Mintlify docs files at the `docs` branch.

This PR also fixes minor formatting errors

### Steps needed before merging
Reference steps: https://github.com/Nixtla/statsforecast/pull/671

## Maintenance instructions
When a new page is added it needs to be added to `mint.json`.

## Test plan
- Install the [Mintlify CLI](https://www.npmjs.com/package/mintlify) (Prerequisite: [node.js](https://nodejs.org/en/download), if you have a mac I like to use [nvm](https://github.com/nvm-sh/nvm)
- Run `bash ./preview-mintlify-docs.bash` to get a preview of the docs locally. Note - the preview will only show `mlforecast` but the Nixtla deployment will be a frankenstein merged version of all 4 docs instances. I created this command so that developers can preview & sanity check their changes before pushing.
